test/Microsoft.AspNet.Mvc.TestCommon/project.json

```diff
@@ -2,5 +2,6 @@
     "version": "6.0.0-*",
     "shared": "**/*.cs",
     "dependencies": {
+        "Microsoft.Framework.WebEncoders.Testing": "1.0.0-*"
```

Comment:
We've previously not placed any dependencies in shared code `project.json` files. Had issues with resolution of these dependencies in the past. But more importantly, the semantics are a bit odd since no compilation happens in this project.